packages/app/src/components/FileMatchButton.cy.tsx

```diff
@@ -6,6 +6,8 @@ import faker from 'faker'
 import { ref } from 'vue'
 const fileMatchButtonSelector = '[data-cy=file-match-button]'
 
+faker.seed(1)
```

Comment:
is it worth even using a faker module vs just removing the dependency and hard-coding a string? 🤔

Reply:
I'd argue it's worth keeping it. I actually like the signal that "this is truly meaningless data, we only care that we get back what we put in" as opposed to a string that should be matched because the component is doing some calculations or something, where it's meaningful that the _specific_ expected string is there.

But we aren't systematic in that kind of usage, so sure, it wouldn't be bad to talk about more.